Documentation/ceph-monitoring.md

```diff
@@ -91,6 +95,7 @@ _Note: This expects prometheus to be pre-installed by the admin._
 
 To enable prometheus alerts after cluster is deployed, run the following commands:  
 ```BASH
+kubectl create -f monitoring-roles.yaml
 cd cluster/examples/kubernetes/ceph/monitoring
 kubectl create -f prometheus-ceph-v<VERSION>-rules.yaml
```

Comment:
Is this step not also required above for enabling alerting during deployment?

Reply:
Also, what should users do if their Ceph version doesn't have rules? There is no prometheus-ceph-v13-rules.yaml

Reply:
> Also, what should users do if their Ceph version doesn't have rules? There is no prometheus-ceph-v13-rules.yaml

Only ceph version >= 14 is supported.

> Is this step not also required above for enabling alerting during deployment?

It is addressed.